test/bdd/features/search.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Searching for assets on the DKG
+  Background: Setup local blockchain, bootstraps and nodes
+    Given the blockchain is set up
+    And 1 bootstrap is running
+
+  Scenario: Search assertion on the network with keywords
+    Given I setup 4 nodes
+    When I call search request on node 1 with result type assertions for the keywords:
```

Comment:
Before performing the search you should first publish the data. So you should add the steps:
When I call publish on node 1 with validAssertion with keywords:
    | keyword 1 | keyword 2 |
    And I wait for last publish to finalize
    And Last publish finished with status: COMPLETED